TripHouse/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="../styles/style.css" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet"/>
+    <title>What do we offer</title>
+</head>
+<body>
+<div class="container-one">
```

Comment:
**Semantic recomendation**
You should use tag `<section>` here for head and a list of items, there are not reasons to use wrapper for heading. It's better to avoid excess wrappers/containers
The` <section>` tag defines a section in a document.
![image](https://user-images.githubusercontent.com/25488411/156405622-de79942f-de2a-4d42-8651-6de5a089d580.png)